.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules/
 .npm/
```

Comment:
Optional, but preferred: could you drop this merge commit and rebase the branch instead?

Reply:
done